content/en/agent/logs/log-transport.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Agent Transport for logs
+kind: documentation
+description: Use the Datadog Agent to collect your logs and send them to Datadog
+further_reading:
+- link: "agent/logs/advanced_log_collection/#filter-logs"
+  tag: "Documentation"
+  text: "Filter logs sent to Datadog"
+- link: "agent/logs/advanced_log_collection/#scrub-sensitive-data-from-your-logs"
+  tag: "Documentation"
+  text: "Scrub sensitive data from your logs"
+- link: "agent/logs/advanced_log_collection/#multi-line-aggregation"
+  tag: "Documentation"
+  text: "Multi-line log aggregation"
+- link: "agent/logs/advanced_log_collection/#tail-directories-by-using-wildcards"
+  tag: "Documentation"
+  text: "Tail directories by using wildcards"
+- link: "agent/logs/advanced_log_collection/#global-processing-rules"
+  tag: "Documentation"
+  text: "Global processing rules"
+---
+
+
+## Default agent behavior
+
+Since Agent v6.19+/v7.19+, the default transport is HTTPS with compression instead of TCP.
+At agent start, if log collection is enabled, the agent will run a HTTPS connectivity test:
+
+* if successfull, then the agent will use HTTPS transport
+* otherwise the agent will fall back to TCP transport
+
+This connectivity test mechanism is only running at agent startup. If the connectivity is failing for both HTTPS and TCP at agent startup time, then the agent will use TCP transport until restart it (and then rerun a new Connectivity test)
+
+To check which transport is used by the agent, run the [agent stauts command][1].
+
+For older version of the agent, TCP is used by default but we strongly recommand you to enforce HTTP transport with compression is you are running v6.14+/v7.14+.
+
+## Enforce a specific transport
+
+The default agent behavior can overwritten by enforcing a specific transport.
+
+{{< tabs >}}
+{{% tab "HTTP" %}}
+
+To enforce HTTP transport, update the Agent's [main configuration file][1] (`datadog.yaml`) with:
+
+```yaml
+logs_enabled: true
+logs_config:
+  use_http: true
+```
+
+To send logs with environment variables, configure the following:
+
+* `DD_LOGS_ENABLED`
+* `DD_LOGS_CONFIG_USE_HTTP`
+
+Note: HTTPS transport is supported with data agent version 6.14+/7.14+
+
+[1]: /agent/guide/agent-configuration-files
+{{% /tab %}}
+{{% tab "TCP" %}}
+
+To enforce TCP transport, update the Agent's [main configuration file][1] (`datadog.yaml`) with:
+
+```yaml
+logs_enabled: true
+logs_config:
+  use_tcp: true
+```
+To send logs with environment variables, configure the following:
+
+* `DD_LOGS_ENABLED`
+* `DD_LOGS_CONFIG_USE_TCP`
+
+By default, the Datadog Agent sends its logs to Datadog over TLS-encrypted TCP. This requires outbound communication over port `10516`.
+
+[1]: /agent/guide/agent-configuration-files
+
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Note: Setting up a [SOCKS5 proxy][4] server will also enforce TCP transport because socks5 proxies are not yet supported in HTTPS with compression.
```

Comment:
As said above, I'd put that in the default agent behavior

Reply:
It is indeed a bit hidden there.

I would actually keep it there but raise it again on the proxy page so that users that leverage the socks5 proxy parameter know that they enforce the TCP transport.